nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -547,11 +566,35 @@ function AddSubscriberDetails(props: DialogProps) {
             state: subscriber.state,
             sub_profile: subscriber.dataPlan,
           },
-        });
-        successCountRef.current = successCountRef.current + 1;
+        };
+        addedSubscribers.push(newSubscriber);
+        // bulk add chunked subscribers
+        if (addedSubscribers.length == SUBSCRIBERS_CHUNK_SIZE) {
```

Comment:
Suggestion: Change above for of loop to include a counter var
`for (const [i, subscriber] of subscribers.entries()) `

Include the below condition within this conditional by adding 
(addedSubscribers.length == SUBSCRIBERS_CHUNK_SIZE) || (i === subscribers.length - 1) {
 // bulk add
    }

Reply:
Done.